### ISSUE TITLE
Port in modules from openstack-ansible

### ISSUE THREAD

**Issue by ansibot** [ansible/ansible#29198]
From @emonty on 2015-03-19T11:42:12Z
##### ISSUE TYPE

Feature Idea
##### COMPONENT NAME

os_stack.py or other openstack modules
##### ANSIBLE VERSION

N/A
##### SUMMARY

@lorin has some great modules in openstack-ansible that have been suggested to be upstreamed in openstack-ansibles/openstack-ansible-modules#40 We should port in the functionality that they have that is not represented here, and then someone should in turn port the code in openstack-ansible/openstack-ansible to use the updated versions here.

Copied from original issue: ansible/ansible-modules-extras#327

**Comment by ansibot** [comment]
From @emonty on 2015-03-19T11:42:12Z

hey @emonty -- I know a lot of stuff has been going on in openstack module land in the past 6 months :) @gregdek and I are just going through and triaging issues -- just checking to see if this is still an idea or if it's been magically implemented by something in the interim :)

**Comment by ansibot** [comment]
From @emonty on 2015-03-19T11:42:12Z

@emonty @Shrews hey...... since i'm sifting through issues this morning.... is this still a thing / idea / etc?

**Comment by ansibot** [comment]
From @emonty on 2015-03-19T11:42:12Z

@robynbergeron yes. we should make sure that we have all of the functionality in upstream ansible that is in the openstack-ansible repos (and also in osad and ursula too) and then ultimately port each of those things to use the upstream modules

**Comment by ansibot** [comment]
From @emonty on 2015-03-19T11:42:12Z

@emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
From @emonty on 2015-03-19T11:42:12Z

@emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @dagnello @dharmabumstead @thingee
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @omgjlk
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by Akasurde** [comment]
@emonty Is this still a valid idea ? needs_info

**Comment by ansibot** [comment]
@emonty This issue is waiting for your response. Please respond or the issue will be closed.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@emonty You have not responded to information requests in this issue so we will assume it no longer affects you. If you are still interested in this, please create a new issue with the requested information.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
